### PR TITLE
RELATED: ONE-5005 Add information about Visualizations Custom Height to SDK doc

### PR DESCRIPTION
**Pull request by NestorEncinas**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/682/ : SUCCESS in 1m 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/431/ : SUCCESS in 9s

File: docs/15_props__chart_config.md [hunk 1]
```diff
@@ -391,3 +393,51 @@ import { InsightView } from "@gooddata/sdk-ui-ext";
 ## Customize tooltips and fonts
 
 To customize tooltips and fonts, [implement a custom visualization](50_custom__create_new_visualization.md).
+
+## Enable Compact Size
+
+The `enableCompactSize` parameter allows the user to set up custom height for the visual components with values that are under the global specifications.
```
Comment (ivan-nejezchleb) [inline docs/15_props__chart_config.md:399]: I would completely ignore custom height. Only purpose of this new flag is to enable more improved responsive behavior of charts. It allows charts to be usefully rendered even in small containers by hiding less important parts. And I would add screenshot some small chart with flag = off (there will be axes visible and area for chart itself will be small) and flag = on (area for chart will be more readable by hiding parts of axes)

File: docs/15_props__chart_config.md [hunk 2]
```diff
@@ -391,3 +393,51 @@ import { InsightView } from "@gooddata/sdk-ui-ext";
...
+        enableCompactSize: {
+            true
+        }
+    }}
+/>
+```
+
+### Responsive Legend
+
+When did is set up will also modify the behaviour of the `Legend` on the visualizations. Adjusting its place accordingly with the visualization height.
```
Comment (ivan-nejezchleb) [inline docs/15_props__chart_config.md:422]: It adjusts position of legend according both dimensions of chart, and also it uses popup for legend content on too small containers

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/690/ : SUCCESS in 1m 00s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/436/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/693/ : SUCCESS in 1m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/438/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/694/ : SUCCESS in 1m 25s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/439/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/695/ : SUCCESS in 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/440/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/697/ : SUCCESS in 1m 36s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/442/ : SUCCESS in 15s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/699/ : SUCCESS in 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/444/ : SUCCESS in 7s

**Comment by evgeniabogranskaya** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/701/ : SUCCESS in 2m 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/445/ : SUCCESS in 14s

**Review by ivan-nejezchleb — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-gate-PR403

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/702/ : SUCCESS in 1m 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/105/ : SUCCESS in 51s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/8/ : SUCCESS in 1m 07s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/196/ : SUCCESS in 20s

**Pull request merged**